default_position :
  position: 
    x: 0.5802925278171578
    y: -0.11142659893152244
    z: 0.05812389832781554
  orientation: 
    x: 0.47355956642108105
    y: 0.5409451607608722
    z: 0.5099788738225833
    w: 0.4722723984684286

default_position_joint_states :
  - -2.739824067854837
  - 3.642702249718859
  - -1.4002863496392823
  - 4.042661172635976
  - 5.157545397608206
  - -1.6057263996584847

use_default_position : True
go_back : 0.1

preplace_transformation :
  position:
    x: 0.0
    y: 0.0
    z: 0.05
  orientation:
    x: 0.0
    y: 0.0
    z: 0.0
    w: 1.0